code/modules/medical/cloning.dm

```diff
@@ -30,6 +30,9 @@
 
 	l_color = "#7BF9FF"
 
+/obj/machinery/cloning/clonepod/empty
```

Comment:
make a map child of clonepod, use NP++ to change the types on the maps and you're done.